tests/containers/bci_prepare.pm

```diff
@@ -67,6 +67,9 @@ sub run {
     my ($self) = @_;
     $self->select_serial_terminal;
 
+    # Wait for any zypper tasks in the background to finish
+    assert_script_run('while true; do sleep 1; if ! ps | grep zypp; then break; fi done', timeout => 300);
```

Comment:
Won't work for two reasons:

1. `ps` only prints processes in the current session, you want something like `ps -ef` instead
2. `ps -ef | grep foo` would find itself, leading to an endless loop

I suggest

```suggestion
    assert_script_run('while pgrep -f zypp; do sleep 1; done', timeout => 300);
```

Reply:
Thanks, PR amended, restarting verification run.